watchfaces/000-default-digital.qml

```diff
@@ -120,6 +126,102 @@ Item {
         }
     }
 
+    Item {
+        id: nightstandMode
+
+        readonly property bool active: mceCableState.connected //ready || (nightstandEnabled.value && holdoff)
```

Comment:
Isn't that supposed to be provided by asteroid-launcher itself ?

Reply:
Ugh, indeed. Same issue as @MagneFire spotted in 010-.
I was not careful enough checking all those PRs :P